arcgis_map_sdk_method_channel/lib/src/method_channel_arcgis_map_plugin.dart

```diff
@@ -137,6 +137,21 @@ class MethodChannelArcgisMapPlugin extends ArcgisMapPlatform {
     ).then((value) => value!);
   }
 
+  @override
+  Future<void> moveCameraToPoints({
+    required List<LatLng> points,
+    required int mapId,
+    double? padding,
+  }) {
+    return _methodChannelBuilder(mapId).invokeMethod<bool>(
+      "move_camera_to_points",
+      {
+        "points": points.map((p) => p.toMap()).toList(),
+        "padding": padding,
+      },
+    ).then((value) => value!);
+  }
+
```

Comment:
No need to force unwrap this value
```suggestion
  @override
  Future<void> moveCameraToPoints({
    required List<LatLng> points,
    required int mapId,
    double? padding,
  }) {
    return _methodChannelBuilder(mapId).invokeMethod<bool>(
      "move_camera_to_points",
      {
        "points": points.map((p) => p.toMap()).toList(),
        "padding": padding,
      },
    );
  }

```